Makefile.toml

```diff
@@ -41,6 +43,8 @@ toolchain = "nightly"
 command = "cargo"
 args = [
   "rustdoc",
+  "-p",
+  "ratatui",
```

Comment:
There should be a way to run `rustdoc` for the workspace members (e.g. `--workspace`)